orchestra/todos/views.py

```diff
@@ -134,7 +135,7 @@ class GenericTodoViewset(ModelViewSet):
     classes can select their own.
     """
     serializer_class = BulkTodoSerializer
-    filter_backends = (filters.DjangoFilterBackend,)
+    filter_backends = (QueryParamsFilterBackend, filters.DjangoFilterBackend)
```

Comment:
if you have QueryParamsFilterBackend, do you need DjangoFilterBackend or filterset_fields?

Reply:
We no longer need `DjangoFilterBackend`. Removed.